api/migrations/20210513022753-create-user.js

```diff
@@ -7,20 +7,6 @@ module.exports = {
         primaryKey: true,
         type: Sequelize.STRING,
       },
-      devBeans: {
```

Comment:
We shouldn't change migrations that have been applied.

Reply:
Will do that for sure next time!

Reply:
Changes to existing migration files will not be applied. Instead, you need to write a new migration.